scripts/exgfs_atmos_awips_20km_1p0deg.sh

```diff
@@ -178,25 +178,25 @@ for GRID in conus ak prico pac 003; do
       # Post Files to ${COMOUT_ATMOS_WMO}
       ##############################
 
-      mv "grib2.awpgfs${fcsthrs}.${GRID}" \
-         "${COMOUT_ATMOS_WMO}/grib2.awpgfs${fcsthrs}.${GRID}"
+      mv "grib2.awpgfs${fcsthr}.${GRID}" \
+         "${COMOUT_ATMOS_WMO}/grib2.awpgfs${fcsthr}.${GRID}"
 
       ##############################
       # Distribute Data
       ##############################
 
       if [[ "${SENDDBN}" == 'YES' || "${SENDAWIP}" == 'YES' ]]; then
           "${DBNROOT}/bin/dbn_alert" NTC_LOW "${NET}" "${job}" \
-				     "${COMOUT_ATMOS_WMO}/grib2.awpgfs${fcsthrs}.${GRID}"
+				     "${COMOUT_ATMOS_WMO}/grib2.awpgfs${fcsthr}.${GRID}"
       else
-          echo "File ${COMOUT_ATMOS_WMO}/grib2.awpgfs${fcsthrs}.${GRID} not posted to db_net."
+          echo "File ${COMOUT_ATMOS_WMO}/grib2.awpgfs${fcsthr}.${GRID} not posted to db_net."
       fi
    elif [[ ${GRID} != "003" ]]; then
-      export FORT11="awps_file_f${fcsthrs}_${GRID}"
-      export FORT31="awps_file_fi${fcsthrs}_${GRID}"
-      export FORT51="grib2.awpgfs_20km_${GRID}_f${fcsthrs}"
+      export FORT11="awps_file_f${fcsthr}_${GRID}"
+      export FORT31="awps_file_fi${fcsthr}_${GRID}"
+      export FORT51="grib2.awpgfs_20km_${GRID}_f${fcsthr}"
 
-      cp "${PARMgfs}/wmo/grib2_awpgfs_20km_${GRID}f${fcsthrs}" "parm_list"
+      cp "${PARMgfs}/wmo/grib2_awpgfs_20km_${GRID}f${fcsthr}" "parm_list"
```

Comment:
We should use `cpreq` when files are required to be able to run, and `cpfs` when output is being copied to `COMOUT`.

Reply:
> cpfs when output is being copied to COMOUT.

Would you suggest we use `cpfs` when copying to `DATA` as well? Example from same script:
```
cp "${COMIN_ATMOS_GRIB_0p25}/gfs.t${cyc}z.pgrb2.0p25.f${fcsthr}" "tmpfile2${fcsthr}"
```

Reply:
Not really intended for use of copy to DATA because the next execution step in DATA is blocked until `cp` finishes.
The intent for use of `cpfs` when copying to `COMOUT` is so that downstream jobs do not get prematurely triggered based on the partial existence of a file.